src/datetime.js

```diff
@@ -1819,10 +1823,12 @@ export default class DateTime {
    * @param {boolean} [opts.includeOffset=true] - include the offset, such as 'Z' or '-04:00'
    * @param {boolean} [opts.extendedZone=false] - add the time zone format extension
    * @param {string} [opts.format='extended'] - choose between the basic and extended format
+   * @param {string} [opts.precision='milliseconds'] - specify desired time precision: 'hours', 'minutes', 'seconds' or 'milliseconds'
```

Comment:
This should mention
- that it will _truncate_ the output
- How it interacts with `suppressSeconds` and `suppressMilliseconds`